.github/workflows/run_tests.yml

```diff
@@ -140,18 +69,19 @@ jobs:
             -DCMAKE_CXX_COMPILER=clang++
             ${{ matrix.add-args || ''}}"
         run: |
-          mkdir build
-          nix develop ./libs/blueprint -c cmake -B build ${{ env.CMAKE_ARGS }} .
+          ls -l -a
+          nix build -L .?submodules=1#checks.x86_64-linux.default

```

Comment:
One command should be enough, fixed that

Reply:
And since blueprint is header-only, `nix build` will actually do nothing, so I guess there is no need to run it at all

Reply:
I think it makes sense to check any anyway. For example, non-test cmake configuration may be broken.

Reply:
I am using the command
```nix build -L .?submodules=1#checks.x86_64-linux.default```
because regular ```nix check``` won't produce tests artifacts.
If I call it like this, it will perform checks

Reply:
It's not the same, nix check does not produce results (they stay in cache and nix build creates a symlink)

Reply:
Yes, test report files. Can you give me an example from any other repo?
I have looked it up from crypto3
https://github.com/NilFoundation/crypto3/blob/f8b34c010be10d8762d3c20ae0e273ae669913e0/.github/workflows/crypto3-testing-linux.yml#L74

Reply:
zkEVM-framework